Makefile

```
@@ -61,13 +61,16 @@ GOOGLEAPI_PROTOS = \
 GOOGLEAPI_PROTOS := $(GOOGLEAPI_PROTOS:%=$(GOPATH)/src/github.com/googleapis/googleapis/%)
 NODE_GOOGLEAPIS = $(GOOGLEAPI_PROTOS:$(GOPATH)/src/github.com/googleapis/googleapis/%.proto=$(NODE_ROOT)/%_grpc_pb.js)
 
+NODE_VTPROT = $(NODE_ROOT)/planetscale/vtprotobuf/vtproto/ext_grpc_pb.js
```

How about naming like below?
```suggestion
NODE_VTEXTJS = $(NODE_ROOT)/planetscale/vtprotobuf/vtproto/ext_grpc_pb.js
```

---

I have fixed it. https://github.com/vdaas/vald-client-node/pull/43/commits/e103aa3e1da67bf4b8f10d3605897841cd23a42f